src/test/librbd/operation/test_mock_SnapshotRemoveRequest.cc

```diff
@@ -148,7 +148,11 @@ TEST_F(TestMockOperationSnapshotRemoveRequest, Success) {
 
   C_SaferCond cond_ctx;
   MockSnapshotRemoveRequest *req = new MockSnapshotRemoveRequest(
-    mock_image_ctx, &cond_ctx, "snap1", snap_id);
```

Comment:
@VictorDenisov I like how it looks right now -- when you have a function call that pushed the params very far to the right such that one or more parameters go way past 80 characters, it looks nicer to bump all the params down to the next line with an indent.